addons/dagr/functions/fnc_menuInit.sqf

```diff
@@ -522,7 +522,7 @@ GVAR(menuRun) = true;
                     (__dsp displayCtrl __mainText) ctrlSetText "Connecting...";
                 };
                 if (ACE_time - GVAR(busyTimer) > 0.6) then {
-                    if (ACE_player hasWeapon "ACE_Vector") then {
+                    if ((ACE_player hasWeapon "ACE_Vector") || {ACE_player hasWeapon "ACE_VectorDay"}) then {
```

Comment:
likewise here, we should avoid hardcoding our vector and use a more flexible approach